js/main.js

```diff
@@ -0,0 +1,80 @@
+document.addEventListener( 'DOMContentLoaded', function(){
+
+    const boxes = document.querySelectorAll('#big-box div');
```

Comment:
przechowujesz referencję do elementu html, według wzorca zmienna powinna się nazywać $boxes

Reply:
Czaję już, wcześniej mi nie działało bo nie zmieniłem we wszystkich miejscach